ColorPicker/ColorPicker/ColorPicker.swift

```diff
@@ -0,0 +1,138 @@
+//
+//  ColorPicker.swift
+//  ColorPicker
+//
+//  Created by Daria Tokareva on 22.06.2021.
+//
+
+import UIKit
+
+protocol ColorPickerDelegate {
+    func colorPicker(colorPicker: ColorPicker, hueValue: CGFloat)
+}
+
+class ColorPicker: UIView {
+
+    var gradientView: UIView!
+    var gradientLayer = CAGradientLayer()
+    var thumbView: UIView!
```

Comment:
1. Рекомендуется сделать дочерние UI-элементы приватными (private), дабы избежать случайные изменения извне. При необходимости можно создать публичные методы для работы с данными UI-элементами. 
2. Для избежания возникновения reference cycle, ссылки на дочерние элементы рекомендуется делать слабыми (weak).

```suggestion
    private weak var gradientView: UIView!
    private var gradientLayer = CAGradientLayer()
    private weak var thumbView: UIView!
```